src/transformer/tests/metadata.test.ts

```diff
@@ -687,6 +687,20 @@ describe('rt:p', it => {
         let params = Reflect.getMetadata('rt:p', exports.C);
         expect(params[0].f).to.contain(F_OPTIONAL);
     });
+    it('emits F_REST for rest ctor param', async () => {
+        let exports = await runSimple({
+            code: `
+                export class A { }
+                export class B { }
+                export class C {
+                    constructor(...hello : A[]) { }
+                }
+            `
+        });
+
+        let params = Reflect.getMetadata('rt:p', exports.C);
+        expect(params[0].f).to.contain(F_REST);
+    });
```

Comment:
I think what you have will also work for methods (ie non-constructors), let's add a test to explicitly test for it.

Reply:
done